notebooks/quickstart.ipynb

```diff
@@ -120,18 +122,115 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 220,
+       "width": 601
+      },
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from libpysal.cg import Point, Chain\n",
+    "import spaghetti\n",
+    "\n",
+    "# create the network\n",
+    "ntw = spaghetti.Network(in_data=Chain([Point([1, 1]), Point([2, 1])]))\n",
+    "\n",
+    "# create the point and snap it to the network\n",
+    "ntw.snapobservations(Point([1.5, 1.1]), \"point\")\n",
+    "\n",
+    "# network nodes and edges\n",
+    "vertices_df, arcs_df = spaghetti.element_as_gdf(ntw, vertices=True, arcs=True)\n",
+    "\n",
+    "# true and snapped location of points\n",
+    "point_df = spaghetti.element_as_gdf(ntw, pp_name=\"point\", snapped=False)\n",
+    "snapped_point_df = spaghetti.element_as_gdf(ntw, pp_name=\"point\", snapped=True)\n",
+    "\n",
+    "# plot the network and point\n",
+    "base = arcs_df.plot(figsize=(10,10), color=\"k\", alpha=0.25, zorder=0)\n",
+    "vertices_df.plot(ax=base, color=\"k\", alpha=1)\n",
+    "kwargs = {\"ax\":base, \"alpha\":0.5, \"zorder\":1}\n",
+    "point_df.plot(color=\"b\", marker=\"x\", **kwargs)\n",
+    "snapped_point_df.plot(color=\"b\", markersize=20, **kwargs)\n",
+    "plt.xlim(.9,2.1); plt.ylim(.8,1.2);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Network creation, observation snapping, and visualization are further reviewed below for an example with empirical datasets available `libpysal`.\n",
+    "\n",
+    "\n",
     "------------------------------\n",
+    "## Empirical Example\n",
     "### 1. Instantiating a `spaghetti.Network` object\n",
     "#### Instantiate the network from a `.shp` file"
```

Comment:
"""
In the following we will walk through an empirical example, visually comparing school locations with a network to crimes committed within the same network. 
"""

Reply:
👍